<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-7J0E1U4" ignored="0" name="generateExtentReports" tests="6" failures="3" timestamp="2022-01-08T22:51:52 CST" time="52.691" errors="0">
  <testcase name="empData" time="4.633" classname="test.GetEmployeeData">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<200>
at test.GetEmployeeData.empData(GetEmployeeData.java:40)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- empData -->
  <testcase name="empData" time="3.019" classname="test.GetEmployeeData">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<200>
at test.GetEmployeeData.empData(GetEmployeeData.java:40)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- empData -->
  <testcase name="postEmployeeData" time="21.69" classname="test.PostEmployeeData"/>
  <testcase name="postEmployeeData" time="6.72" classname="test.PostEmployeeData"/>
  <testcase name="putEmployeeData" time="14.503" classname="test.PutEmployeeData">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at test.PutEmployeeData.putEmployeeData(PutEmployeeData.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- putEmployeeData -->
  <testcase name="deleteEmployee" time="0.751" classname="test.DeleteEmployeeData"/>
</testsuite> <!-- generateExtentReports -->
